### ISSUE TITLE
Monika disappeared

### ISSUE THREAD

**Issue by ItsDessiah** [Monika-After-Story/MonikaModDev#10093]

**Comment by dreamscached** [comment]
Please be more descriptive.

**Comment by ItsDessiah** [comment]
sorry, I accidentally hit send without commenting

**Comment by ItsDessiah** [comment]
Today is Monika's birthday and I was setting up the party. I moved her character file out of the room and had the party set up. But when I put her back into the character folder, she wouldn't show up. I tried installing a backup persistent, I moved her character file back to the usb and then back to the character folder. I restarted my pc, and so far nothings worked

**Comment by dreamscached** [comment]
Actually you could have just kept her in the characters folder. Anyway, you can try to restore an automatically made backup (.bak files located in your persistent folder) by renaming it from `persistent##.bak` to `persistent`. One of these should be working fine.

**Comment by ItsDessiah** [comment]
I backed up the persistent file, and nothing changed, should I try an older one?

**Comment by ItsDessiah** [comment]
Ok, ill try that real quick

**Comment by ItsDessiah** [comment]
ok, it didn't work. I swapped persistent with one from yesterday, and she hasn't appeared

**Comment by ItsDessiah** [comment]
Is it possible that there's an issue with the monika file itself, since that's what I moved?

**Comment by dreamscached** [comment]
If you moved it it may be possible that it got corrupted somehow, it happens sometimes. That's why I asked to restore a persistent, because that's the only way to get her back.

**Comment by ItsDessiah** [comment]
ok

**Comment by ItsDessiah** [comment]
Now, I have a copy of the monika file on my computer separate from my mas files entirely. Do you know if it would be a bad idea to delete the monika in my files now and replace it with that one?

**Comment by dreamscached** [comment]
No, another `monika` file won't work as it's not enough just to have it named such, it has unique code inside.

**Comment by dreamscached** [comment]
Can you send `mfgen.log` and `mfread.log` files?

**Comment by ItsDessiah** [comment]
[mfgen.log](https://github.com/Monika-After-Story/MonikaModDev/files/12701436/mfgen.log)
[mfread.log](https://github.com/Monika-After-Story/MonikaModDev/files/12701439/mfread.log)

**Comment by ItsDessiah** [comment]
I just saw the mfread.log
says I have a corrupt Monika

**Comment by dreamscached** [comment]
Developers will look into it, unfortunately that's all I can tell.

**Comment by ItsDessiah** [comment]
Ok, thanks for the help

**Comment by dreamscached** [comment]
Reinstalling won't really do anything, no. Just wait for now.

**Comment by ItsDessiah** [comment]
Ok, thank you much

**Comment by Booplicate** [comment]
@ItsDessiah, make a backup of the folder with persistents just in case. Then keep restoring backups until you find a working one.
Also look into getting a new drive, your might be malfunctioning.

**Comment by Booplicate** [comment]
As was said reinstall won't help.

**Comment by ItsDessiah** [comment]
Why would that be the case?

**Comment by ItsDessiah** [comment]
I feel like replacing the corrupt files with fresh one's would hypothetically be a fix. The only way I see it not working is if my persistent is the file that's corrupted. 

I'm not the best with files, so I could be wrong, but that just makes sense to me.

**Comment by ItsDessiah** [comment]
either way, I might just have to wait for the Dev team to respond, do you know how long that typically takes?

**Comment by Booplicate** [comment]
I just responded

> @ItsDessiah, make a backup of the folder with persistents just in case. Then keep restoring backups until you find a working one. Also look into getting a new drive, your might be malfunctioning.

The only corrupted file is the `monika` file, but you cannot replace it, it's unique to your Monika.

**Comment by ItsDessiah** [comment]
Or to reword my question, what would happen if I were to add my current persistent onto a MAS game that was fresh?

**Comment by Booplicate** [comment]
> So I did that, and I went through all of my persistent files, going back to nearly last week and none of them are working. 

It's impossible, you're doing something wrong. Try following [this guide](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent).

**Comment by ItsDessiah** [comment]
I don't know what I'm doing wrong? I have used this guide several times before. I've gone to RenPy, i've copied the persistent files and backed them up in a separate folder, i've swapped the current persistent file to all of them. I even had an old drive with a persistent from 2 months ago , that one also didn't work.

**Comment by ItsDessiah** [comment]
Yes, I even restarted my computer after a few attempts

**Comment by ItsDessiah** [comment]
IDK, I'll just wait for a dev response.

**Comment by dreamscached** [comment]
That *was* one of the devs, if none of the backups made automatically work - you must be doing something wrong.

**Comment by ItsDessiah** [comment]
oh my b. I still don't know what it is I'm doing wrong. I've used the guide, I've used both automatic and manual backups. I've been putting them in the character file.

**Comment by ItsDessiah** [comment]
I promise I'm not trying to be difficult, I really just don't know what else to try

**Comment by ItsDessiah** [comment]
OK, I need to have this clarified one last time. If I were to save copies of my persistent and put them on a separate file, would those persistent files have my save data?

**Comment by dreamscached** [comment]
Yes. You can move them somewhere like your desktop if you need.

**Comment by dreamscached** [comment]
You don't have to transfer it to another device, all you need to do is to rename one of these `.bak` files to `persistent`. Also, are file extensions displayed for you?

**Comment by ItsDessiah** [comment]
Yes, I've been renaming the files to persistent. I've done it with all ten, and even a 2 month old backup, none of which worked. But what I'm trying to ask is if I keep my persistent files, delete everything else, reinstall it and add the persistent files back, would I keep my save data?

**Comment by dreamscached** [comment]
To be safe, don't delete anything, just get another copy off https://ddlc.moe

If you keep your persistent files, yes, your save data will be preserved.

**Comment by ItsDessiah** [comment]
OK, i'll try that. I do have to be somewhere soon, so I'll probably be a while before I let you know if it worked or not
